sw/device/lib/dif/dif_otbn.h

```diff
@@ -255,6 +256,17 @@ dif_otbn_result_t dif_otbn_is_busy(const dif_otbn_t *otbn, bool *busy);
 dif_otbn_result_t dif_otbn_get_err_bits(const dif_otbn_t *otbn,
                                         dif_otbn_err_bits_t *err_bits);
 
+/**
+ * Get the instruction counter value.
```

Comment:
Nit: comments must be in indicative mood, i.e., `Gets the instruction counter value.`

(The rest of this file seems to be in violation of the style guide but that's a cleanup for later.)